### PR TITLE
[AAE-12184] add possible environment to apps process cloud service

### PR DESCRIPTION
**Pull request by DiogoABastos**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Apps process cloud service did not account for possible environment

**What is the new behaviour?**
Apps process cloud service accepts possible environment to be sent as parameter
This change was done to support changes in admin app.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services-cloud/src/lib/app/services/apps-process-cloud.service.ts [hunk 1]
```diff
@@ -80,8 +80,11 @@ export class AppsProcessCloudService {
             );
     }
 
-    private getApplicationUrl(): string {
-        return `${this.appConfigService.get('bpmHost')}/deployment-service/v1/applications`;
+    private getApplicationUrl(envKey?: string): string {
+        let contextRoot = this.appConfigService.get('bpmHost', '');
+        contextRoot = 'https://hxps.dev.hxp.hyland.com';
```
Comment (DenysVuika) [inline lib/process-services-cloud/src/lib/app/services/apps-process-cloud.service.ts:85]: you are hardcoding a real server address to the public open source project ;)

File: lib/process-services-cloud/src/lib/app/services/apps-process-cloud.service.ts [hunk 2]
```diff
@@ -80,8 +80,11 @@ export class AppsProcessCloudService {
...
+        contextRoot = contextRoot.split('://').join(`://${envKey ? 'hxps-' + envKey + '.' : ''}`);
```
Comment (DenysVuika) [inline lib/process-services-cloud/src/lib/app/services/apps-process-cloud.service.ts:86]: is it okay that the ADF lib now tightly bound to the "hxps-" ?

Comment (DiogoABastos) [reply lib/process-services-cloud/src/lib/app/services/apps-process-cloud.service.ts:86]: It shouldn't. Maybe I can add the hxps as part of the string that is sent to ADF, what do you think?

**Comment by DiogoABastos** [comment]
No longer required

**Pull request closed without merging**